tests/e2e/specs/gscan.js

```diff
@@ -22,15 +22,14 @@ describe('GScan component', () => {
       .get('.c-gscan-workflow')
       .should('have.length', 1)
   })
+
   it('should filter by workflow name', () => {
-    cy.visit('/#/')
```

Comment:
@MetRonnie could you explain this diff.

Reply:
As this test can be carried out on the state of the UI after the success of the previous test, it is quicker not to reload the page

Reply:
So Cypress maintains its state from one test to the next? Is this just for "it" declarations or also "describe"? Will this test still work if it is change to "it.only"?

Reply:
It seems to me it maintains the state during a whole test file. I think you are only allowed 1 top-level `describe`, but any sub-`describe`s probably have the same state. Unless you do `cy.visit()`. I suspect it won't work if you do `it.only`, but if you're doing that you could just as easily add `cy.visit()` as well?

Reply:
Unfortunately I don't think you're allowed `cy.visit()` outside an `it()`. There are some tests where it is useful to have inter-dependent tests because the behaviour relies on past interactions, particularly with the mutations menu (e.g. checking it is collapsed when you open it after having closed it while expanded), for those I think it would be somewhat a pain to have to duplicate getting it into a particular state when the earlier tests already got it into that state.

I don't think `--watch` is a problem though? When I run the e2e tests it hot reloads when I save the file just fine

Reply:
> The new test has uncovered a bug: if you run it you will see that the menu doesn't appear if you click on a workflow icon that isn't the current workflow

This bug is more specific than that - It only occurs if you try to press a workflow icon in the dashboard. You can in fact control other workflows as long as the current view isn't the dashboard!